vite-project-kabinet/src/Content/PaginationNav.jsx

```diff
@@ -0,0 +1,34 @@
+import arrowRight from "./assets/arrow_right.svg"
+import arrowLeft from "./assets/arrow_left.svg"
+
+export const PaginationNav = ({currentPage, totalPages, handlePageChange}) => {
+
+    return (
+        <div className="pages_pc">
+            {currentPage > 1 &&
+                <button
+                    className="button-pc"
+                    onClick={() => handlePageChange(currentPage - 1)}
+                    disabled={currentPage === 1}
+                >
+                    <img src={arrowLeft} alt="šípka vľavo"/>
+                </button>
+            }
+
+            <ul className="pages__numbers">
+                <li>
+                    <button className="button-pc">{currentPage}</button>
+                </li>
+            </ul>
```

Comment:
Proč je tohle odrážkový seznam, když to má jen jednu položku?

Reply:
Preto je to takto, aby sa tam v budúcnosti mohlo prípadne namapovať viac tých stránok. Ja si ale myslím, že klient bude so súčasným riešením spokojný.